compiler/ballerina-parser/src/test/resources/types/tuple-type/tuple_type_assert_04.json

```diff
@@ -1222,12 +1343,51 @@
                       "children": [
                         {
                           "kind": "IDENTIFIER_TOKEN",
-                          "value": "j"
+                          "value": "a"
                         }
                       ]
                     }
                   ]
                 },
+                {
+                  "kind": "EQUAL_TOKEN",
+                  "isMissing": true,
+                  "hasDiagnostics": true,
+                  "diagnostics": [
+                    "ERROR_MISSING_EQUAL_TOKEN"
```

Comment:
In here,
Input: ` [string % int a] j;`
Previous Recovery: `[string  MISSING[,] INVALID[%] int  MISSING[,]a] j;`
New Recovery:`[string  MISSING[,] INVALID[%] int  MISSING[]]a MISSING[ INVALID[]] =]j;`

Seems the parser can't detect the missing comma between `int` and `a` now.